javaslang/src/main/java/javaslang/collection/TreeMultimap.java

```diff
@@ -194,7 +205,7 @@ private Builder(ContainerType containerType, SerializableSupplier<Traversable<?>
 
     /**
      * Narrows a widened {@code HashMultimap<? extends K, ? extends V>} to {@code HashMultimap<K, V>}
-     * by performing a type safe-cast. This is eligible because immutable/read-only
+     * by performing a type-safe cast. This is eligible because immutable/read-only
```

Comment:
Thanks, will try that - still searching the right tool. Currently I use GitUp for Mac - nice and simple.